common/params.cc

```diff
@@ -129,6 +129,7 @@ std::unordered_map<std::string, uint32_t> keys = {
     {"HasAcceptedTerms", PERSISTENT},
     {"IMEI", PERSISTENT},
     {"InstallDate", PERSISTENT},
+    {"IsDeviceAwake", CLEAR_ON_MANAGER_START},
```

Comment:
don't think we need a new param for this, IsOffroad/IsOnroad should be sufficient